packages/driver-interface/services/bot.js

```diff
@@ -1,22 +1,43 @@
 const amqp = require('./amqp')
-const { getVehicle, addVehicle } = require('./cache')
+const {
+  getVehicle,
+  addVehicle,
+  setInstructions,
+  getInstructions,
+} = require('./cache')
+const { getBooking } = require('./cache')
+
+const messaging = require('./messaging')
 
 const onLogin = (vehicleId, ctx) => {
-  const oldVehicle = getVehicle(vehicleId)
-  const telegramId = ctx.botInfo.id
-  ctx.metadata.setId(telegramId, vehicleId)
-  if (oldVehicle && oldVehicle.telegramId) {
+  const currentVehicle = getVehicle(vehicleId)
+  if (!currentVehicle) return ctx.reply('Någonting gick fel..')
```

Comment:
Maybe change this to: "Kunde inte hitta det förarIdt" or similar

Reply:
Updated!